### DISCUSSION TITLE
windows version

### DISCUSSION THREAD

**Discussion by AmA-sud** [madMAx43v3r/chia-plotter#115] [Q&A]
hi there,

recently have see a video on youtube ting software, I was wondering are you planning on compiling a windows version of it ?

**Comment by grayfallstown** [comment]
Unofficial: https://github.com/stotiks/chia-plotter/releases

**Reply by AmA-sud** [reply]
its not working!

**Reply by SebMoore** [reply]
The windows version uses the old commands, not the new ones. That might be why.
![image](https://user-images.githubusercontent.com/74476668/121483865-01010c00-ca12-11eb-88a4-90f93284d7c0.png)

**Reply by AmA-sud** [reply]
it just gives this and then dies 
Number of Plots: 1
Process ID: 10152
Number of Threads: 30
Number of Buckets: 2^7 (128)

**Reply by AmA-sud** [reply]
now it just keeps saying
Invalid <pool_key>: 00 (needs to be 48 bytes)

**Reply by SebMoore** [reply]
Can you paste here the exact command you typed in to run the program?

**Reply by AmA-sud** [reply]
.\chia_plot.exe create -p xxxxxx -f xxxxxx -b 12288  -r 30 -u 128 -n 1 -t D:\ -2 E:\ -d Z:\

**Reply by SebMoore** [reply]
Yep so again, you need to use the old command syntax for this windows version. That will look like this:
`chia-plot.exe  poolkey farmerkey D:\ E:\ 30 7`
Replace poolkey with your pool key, and farmerkey with your farmer key. This is because the windows version was compiled about 20 hours ago when this new -u -n whatever stuff wasn't done yet. Note: the windows version does not support final copy, you'll have to move the plot files to your Z drive by yourself. Also, num buckets in the old version was a log number so 7 = 2^7 buckets = 128 buckets. That's why there's a 7 in the command.

**Reply by AmA-sud** [reply]
its still the same issue, are there dependencies that I should install on  windows

**Reply by soundy83** [reply]
Yeah I am also having the same issue.
I am using the 'old' command as suggested.
But I just get the following:

30 7
Number of Threads: 30
Number of Sort Buckets: 2^7 (128)

It creates a file in each of the Temp storage locations and then just returns to the prompt.

**Reply by AmA-sud** [reply]
this is the windows event log:

Faulting application name: chia_plot.exe, version: 0.0.0.0, time stamp: 0x60c180f8
Faulting module name: ucrtbase.dll, version: 10.0.17763.1490, time stamp: 0x48ac8393
Exception code: 0xc0000409
Fault offset: 0x000000000006e77e
Faulting process id: 0xf84
Faulting application start time: 0x01d75de24757acdb
Faulting application path: C:\users\administrator\desktop\madmax\chia_plot.exe
Faulting module path: C:\Windows\System32\ucrtbase.dll
Report Id: 4775de14-f764-4f86-8b12-c59316719e5a
Faulting package full name: 
Faulting package-relative application ID:

**Reply by SebMoore** [reply]
Ok I see this is a wider issue now - see bottom of #74, #126. I don't know of a solution as of now.

**Reply by prodchia** [reply]
The windows version seems to be working for me. The only weird thing I noticed is that on a 40 cores machine, -r 16 options takes over the full 40 cores.

**Reply by zxmtangfan** [reply]
Your computer doesn't support it. Is it the main board of x79

**Reply by deannamnguyen** [reply]
> to be working for me. The only weird thing I noticed is that on a 40 cores machine, -r 16 options takes over the full 40 cores.

could you please share how to get the windows version to work? what is the command you use?

**Reply by deannamnguyen** [reply]
tried with both command, all end up killing process and gave this.
Final Directory: G:\
Number of Plots: 20
Process ID: 5936
Number of Threads: 20
Number of Buckets: 2^7 (128)

**Reply by prodchia** [reply]
> Your computer doesn't support it. Is it the main board of x79

I didn't do anything special. Just used the following command

c:\> chia_plot.exe create -p xxxxxx -f xxxxxx -b 12288 -r 30 -u 128 -n 1 -t D:\ -2 E:\ -d Z:\

As someone suggested, there could be a compatability issue with your hardware

**Reply by delucca** [reply]
In the meantime, you could use a Docker container, as explained in #102

**Comment by zxmtangfan** [comment]
https://github.com/stotiks/chia-plotter/releases/tag/v0.0.3-alpha




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年6月11日(星期五) 凌晨0:24
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [madMAx43v3r/chia-plotter] windows version (#115)





  
为我工作。我注意到的唯一奇怪的事情是在 40 核机器上，-r 16 选项接管了全部 40 核。
  
你能分享一下如何让windows版本工作吗？你使用的命令是什么？
 
—
您收到此消息是因为您发表了评论。
直接回复此邮件，在 GitHub 上查看，或取消订阅。

**Reply by deannamnguyen** [reply]
I already try this release but no change

**Reply by giordanidev** [reply]
> I already try this release but no change

is your MB chip x79?

i think its not working on those as it's not working here too. i have a x79 mb.

**Comment by zxmtangfan** [comment]
Your computer doesn't support it. Is it the main board of x79







Your computer doesn't support it. Is it the main board of x79












Your computer doesn't support it. Is it the main board of x79












Your computer doesn't support it. Is it the main board of x79









------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年6月11日(星期五) 凌晨0:26
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [madMAx43v3r/chia-plotter] windows version (#115)





 
尝试使用这两个命令，最终都终止了进程并给出了这个。
 最终目录：G：
 图数：20
 进程 ID：5936
 线程
 数：20桶数：2^7 (128)
 
—
您收到此消息是因为您发表了评论。
直接回复此邮件，在 GitHub 上查看，或取消订阅。

**Comment by delucca** [comment]
In the meantime, you could use a Docker container, as explained in #102

**Comment by scerbera** [comment]
Whoever made the windows build, many thanks. here is a screen grab of my 3990x running 6 in parallel.
![image](https://user-images.githubusercontent.com/85704643/121680205-2e50c700-cab1-11eb-8db8-b69ec98b8031.png)

**Reply by brunobrdo** [reply]
6 in parallel using madmax plotter?
What's your setup (hardware)?

**Reply by scerbera** [reply]
3990x 64 core threadripper. using a nvme drive for temps and final. if you run max threads in phase 1 (on this machine), the cpu utilisation drops off when you get to phase 2-4 so you go from 100% to 60% ish. by limiting to 8 threads it will keep at 8 all the way through so can run a few in parallel.

Still experimenting but should easily better my previous plotting on this machine.

**Reply by brunobrdo** [reply]
Nice, I think we have similar hardwares.
I have Threadripper 3990x 64/128, 128GB RAM and (2+2+1) TB NVMe (Firecuda 520 series).
So... limiting 8 threads/plot you are running 6 in parallel.
Sounds very good.
But, how long to complete each plot?

**Reply by scerbera** [reply]
I'm going to give it 12 hours to get settled in to the correct timing and Ill let you know
